build.sbt

```diff
@@ -44,6 +45,36 @@ lazy val toolkitTest = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     mimaPreviousArtifacts := Set()
   )
 
+lazy val toolkitTesting = project
+  .in(file("toolkit-testing"))
+  .settings(
+    name := "toolkit-testing",
+    Test / test := (Test / test).dependsOn(toolkit.jvm / publishLocal).value,
```

Comment:
It doesn't have to be this PR, but eventually I'd like to publish the JS and Native toolkits and test those as well.

Specifically, we seem to have an issue sometimes where the toolkit requires JS or Native versions newer than what the latest scala-cli is shipping by default, I wonder if we should try to avoid merging those ...
- https://github.com/typelevel/toolkit/issues/49

Reply:
Hmm, this can be done easily, but using scala-cli as an external process, a thing that will solve the `System.exit(1)` too.

That's another point in favor of this solution 🤔